addons/purchase/models/purchase.py

```diff
@@ -899,7 +899,7 @@ class PurchaseOrderLine(models.Model):
     currency_id = fields.Many2one(related='order_id.currency_id', store=True, string='Currency', readonly=True)
     date_order = fields.Datetime(related='order_id.date_order', string='Order Date', readonly=True)
     product_packaging_id = fields.Many2one('product.packaging', string='Packaging', domain="[('purchase', '=', True), ('product_id', '=', product_id)]", check_company=True)
-    product_packaging_qty = fields.Float('Packaging Quantity')
+    product_packaging_qty = fields.Float('Pkg Qty')
```

Comment:
Heuuuu. Don't do that. String should be the more complete as possible. If it takes too much in a view then overwrite the string in the view rather than here